RecoBTag/ONNXRuntime/plugins/ParticleTransformerAK4ONNXJetTagsProducer.cc

```diff
@@ -151,24 +139,21 @@ void ParticleTransformerAK4ONNXJetTagsProducer::get_input_sizes(
     const reco::FeaturesTagInfo<btagbtvdeep::ParticleTransformerAK4Features> taginfo) {
   const auto& features = taginfo.features();
 
-  unsigned int n_cpf = features.c_pf_features.size();
-  unsigned int n_npf = features.n_pf_features.size();
-  unsigned int n_vtx = features.sv_features.size();
+  n_cpf_ = features.c_pf_features.size();
+  n_npf_ = features.n_pf_features.size();
+  n_sv_ = features.sv_features.size();
 
-  n_cpf_ = std::max((unsigned int)1, n_cpf);
-  n_npf_ = std::max((unsigned int)1, n_npf);
-  n_sv_ = std::max((unsigned int)1, n_vtx);
+  n_cpf_ = std::clamp(n_cpf_, (unsigned int)1, (unsigned int)25);
+  n_npf_ = std::clamp(n_npf_, (unsigned int)1, (unsigned int)25);
+  n_sv_ = std::clamp(n_sv_, (unsigned int)1, (unsigned int)5);
```

Comment:
25, 25, and 5 should be moved to named constants in the `parT` namespace in `tensor_configs.h`, so that they can be shared with the SONIC version rather than copying the hardcoded values.